modules/lightning_features/lightning_media/config/install/core.entity_view_display.media.image.embedded.yml

```diff
@@ -0,0 +1,32 @@
+uuid: 52579bba-569e-497d-9400-b3fe7c6a752c
```

Comment:
We don't want the uuid here, do we? It does make me think that we might be able to leverage UUIDs in our config files in whatever our long term plan is for updates though. hmmm.